server/routers/capturesRouter.ts

```diff
@@ -71,7 +71,8 @@ router.get(
       query,
       Joi.object().keys({
         grower_account_id: Joi.string().uuid(),
-        organization_id: Joi.array(),
```

Comment:
Is this a deliberate change? `organization_id` should be an array to allow us to query multiple sub-organizations in one go.

Reply:
This is probably a mistake that I made when I add session_id field. I have pushed a fix for it

Reply:
@maxt0214 We're starting to add more detail to the validation.  Can you please update to this ...
`organization_id: Joi.array().items(Joi.string().uuid()),
        session_id: Joi.string().uuid(),`
       
... and confirm your code is up-to-date, then resubmit for review.  Thanks!